docs/guides/external-node/docker-compose-examples/mainnet-external-node-docker-compose.yml

```diff
@@ -22,12 +20,7 @@ services:
       postgres:
         condition: service_healthy
     volumes:
-      - type: bind
-        source: volumes/mainnet-external-node-state-keeper
-        target: ./db/ext-node/state_keeper
-      - type: bind
-        source: volumes/mainnet-external-node-merkle-tree
-        target: ./db/ext-node/lightweight
+      - rocksdb:/usr/src/zksync/db
```

Comment:
Are you sure that this is the path used for the RocksDB instances? AFAICT, `/usr/src/zksync` is set as the `WORKDIR` only for the builder container in [the EN `Dockerfile`](https://github.com/matter-labs/zksync-era/blob/main/docker/external-node/Dockerfile), so the default `/` workdir should be used for the main container (which seems to be confirmed by the output of `docker run --rm --entrypoint=/bin/sh matterlabs/external-node:e03e201-1712331604214-linux-amd64 -c 'ls .'`).

Reply:
nice catch, I haven't noticed it being a builder container